src/Penrose/Style.hs

```diff
@@ -339,9 +340,10 @@ block :: Parser [Stmt]
 block = stmt `sepEndBy` newline'
 
 stmt :: Parser Stmt
-stmt = tryChoice [assign, override, delete]
+stmt = tryChoice [assign, anonAssign, override, delete]
 
-assign, override, delete :: Parser Stmt
+anonAssign, assign, override, delete :: Parser Stmt
+anonAssign = AnonAssign <$> expr
```

Comment:
Depending on the requirements for the language changes, we might not want all exprs to be valid here. It’s easy to write a new function that’s just a simple trychoice list (say, anonExprs) to group valid expressions for anonassign

Reply:
Oh yeah, I forgot to check that only certain expression forms be allowed. I'll try that